specification/columns/usagequantity.md

```diff
@@ -1,27 +1,27 @@
 # Usage Quantity
 
-The Usage Quantity represents the volume of a given SKU associated with a [*resource*](#glossary:resource) or [*service*](#glossary:service) used or purchased, based on the [Usage Unit](#usageunit). Usage Quantity is often derived at a finer granularity or over a different time interval when compared to the [Pricing Quantity](#pricingquantity) (complementary to [Pricing Unit](#pricingunit)), and focuses on *resource* and *service* consumption, not pricing and cost.
+The Usage Quantity represents the volume of a given SKU associated with a [*resource*](#glossary:resource) or [*service*](#glossary:service) used or purchased, based on the [Usage Unit](#usageunit). Usage Quantity is often derived at a finer granularity or over a different time interval when compared to the [Pricing Quantity](#pricingquantity) (complementary to [Pricing Unit](#pricingunit)) and focuses on *resource* and *service* consumption, not pricing and cost.
 
-UsageQuantity MUST be present in the billing data. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat). The value MAY be negative in cases where [ChargeSubcategory](#chargesubcategory) is "Refund". This column MUST NOT be null if [SkuPriceId](#skupriceid) is not null and MUST be null if SkuPriceId is null.
+UsageQuantity column MUST be present in the billing data when the provider supports the measurement of usage. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements. The value MAY be negative in cases where [ChargeSubcategory](#chargesubcategory) is "Refund". This column MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" or "Purchase" and MAY be null for other ChargeCategory values.
```

Comment:
This feels a bit abstract. Can we bind it to ChargeCategory = Usage or when there's a difference between pricing and usage?
```suggestion
UsageQuantity column MUST be present in the billing data when the provider supports usage-based charges where [ChargeCategory](#chargecategory) is "Usage" or when [PricingQuantity](#pricingquantity) does is based on [block pricing](#glossary:block-pricing) or measured in a different unit. This column MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements. The value MAY be negative in cases where [ChargeSubcategory](#chargesubcategory) is "Refund". This column MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" or "Purchase" and MAY be null for other ChargeCategory values.
```

Reply:
From my practitioner perspective, if a provider omits the UsageQuantity column because it's the same as PricingQuantity, I would not be able to make the distinction with the case where the provider does not support the measurement of usage. So, in the case the UsageQuantity is the same as PricingQuantity, I would prefer UsageQuantity is present and populated with the same as PricingQuantity.

Moreover, the presence of the column should not be related to the value of the ChargeCategory, because this value may differ between records.

Reply:
> This feels a bit abstract. Can we bind it to ChargeCategory = Usage or when there's a difference between pricing and usage?

Agree that it needs to not be abstract - and open to make it clearer. 

However, as described, anyone with block pricing regardless of if the provider bills based on 'usage' would have to include this column. The usage* columns were recently broadened to include usage and purchase instead of just usage (which I'm personally not a fan of). In that world, how do we write it in a way that it's not a barrier for entry (and a confusion for customers) when usage* columns are included with values? Might reach out to you both in slack to see if we can drive consensus on this.

Reply:
> as described, anyone with block pricing regardless of if the provider bills based on 'usage' would have to include this column

That was my goal 🙃 When there's a difference between usage and pricing, you must have both. That feels like the same as including CommitmentDiscount* columns when commitment discounts are supported.

Fwiw, I agree with Marc that it's best to always have UsageQuantity since that's necessary to do usage-based analysis without block pricing across providers. But I'm happy to defer to practitioners.

Reply:
Not sure I understand. Many providers don't support usage-based billing but might have block pricing (and bill up-front for what you bought). They would now have to provide usage* columns which would provide completely inaccurate information. This is the reason we moved this to non-essential based on the discussion in the task forces.

Reply:
If there's an issue to discuss here, lets create a different GH issue and work it. As is, its just replacing the reference to a non-mandatory column. Some of these will get updated with the ChargeClass column nuance anyway.